extensions/amp-story/0.1/amp-story.js

```diff
@@ -1282,7 +1337,6 @@ export class AmpStory extends AMP.BaseElement {
     if (this.bookend_.isActive()) {
       // Dispaching event instead of calling method directly so that all
       // listeners can respond.
-      dispatch(this.element, EventType.CLOSE_BOOKEND);
```

Comment:
I don't think you should remove this :)

Reply:
How did that happen?!

Reply:
My guess is cut/paste for `dispatch`? ;)